test/board_test.rb

```diff
@@ -24,4 +26,25 @@ def test_is_validates_coordinates
     assert_equal false, @board.valid_coordinate?("E1")
     assert_equal false, @board.valid_coordinate?("A22")
   end
+
+  def test_returns_valid_placement_length
+    assert_equal false, @board.valid_placement?(@cruiser, ["A1", "A2"])
+    assert_equal false, @board.valid_placement?(@submarine, ["A2", "A3", "A4"])
+  end
+
+  def test_valid_placement
+    assert_equal false, @board.valid_placement?(@cruiser, ["A1", "A2", "A4"])
```

Comment:
I like these tests that shows what should happen when the coordinates aren't consecutive.